src/components/User/Username.tsx

```diff
@@ -1,24 +1,42 @@
 import React from 'react'
+
 import Avatar from 'decentraland-gatsby/dist/components/User/Avatar'
-import { Blockie } from 'decentraland-ui/dist/components/Blockie/Blockie'
-import { Address } from 'decentraland-ui/dist/components/Address/Address'
 import { Link } from 'decentraland-gatsby/dist/plugins/intl'
+import { Address } from 'decentraland-ui/dist/components/Address/Address'
+
 import useProfile from '../../hooks/useProfile'
 import locations from '../../modules/locations'
 
-const Username = ({ address }: { address: string }) => {
+import './Username.css'
+
+interface Props {
+  address: string
+  avatarSize?: 'big' | 'small' | 'mini' | 'tiny' | 'medium' | 'large' | 'huge' | 'massive' | 'full'
+  linked?: boolean
+  showName?: boolean
+}
+
+const Username = ({ address, avatarSize, linked, showName }: Props) => {
```

Comment:
`showName` can be defaulted to true so you can avoid setting this prop in every component as it is is always set as true.